CMakeLists.txt

```diff
@@ -1216,6 +1216,14 @@ if(NOT CONFIG_BUILD_NO_GAP_FILL)
   set(GAP_FILL "--gap-fill;0xff")
 endif()
 
+if(CONFIG_OUTPUT_PRINT_MEMORY_USAGE)
+  # @Intent: Use the toolchain bintools method for printing memory usage
+  toolchain_bt_output_print_mem_usage(
```

Comment:
Don't use bt as a shorthand for bintools.

Reply:
Ok, what would you suggest instead of bt? bintools perhaps, thus making the macro named toolchain_bintools_output_print_mem_usage?

Reply:
yes, either that or just bintools_output_print_mem_usage, or bintools_print_mem_usage.

Isn't output redundant?

Reply:
Lets go for bintools_print_mem_usage since both toolchain and output is not really needed. I agree that output is redundant, and it was only added because the configuration option included output and print, but better to simplify that.